uaclient/security.py

```diff
@@ -701,8 +701,10 @@ def _handle_released_package_fixes(
     binary_pocket_pkgs: "Dict[str, List[str]]",
     pkg_index: int,
     num_pkgs: int,
-) -> bool:
+) -> "Tuple[bool, List[str]]":
     """Handle the packages that could be fixed and have a released status."""
+    upgrade_status = True
+    not_fixed_pkgs = []
```

Comment:
I think we should substitute not_fixed for unfixed throughout this file and comments.